packages/tunnel/src/commands/deploy/index.ts

```diff
@@ -0,0 +1,86 @@
+import { existsSync } from 'node:fs';
+import path from 'node:path';
+
+import { isValidUrl } from '@logto/core-kit';
+import chalk from 'chalk';
+import type { CommandModule } from 'yargs';
+
+import { consoleLog } from '../../utils.js';
+
+import { type DeployCommandArgs } from './types.js';
+import { deployToLogtoCloud } from './utils.js';
+
+const tunnel: CommandModule<unknown, DeployCommandArgs> = {
+  command: ['deploy'],
+  describe: 'Deploy your custom UI assets to Logto Cloud',
+  builder: (yargs) =>
+    yargs.options({
+      'app-id': {
+        alias: ['id'],
+        describe: 'ID of your Logto M2M application',
+        type: 'string',
+      },
+      'app-secret': {
+        alias: ['secret'],
+        describe: 'Secret of your Logto M2M application',
+        type: 'string',
+      },
+      endpoint: {
+        describe:
+          'Logto endpoint URI that points to your Logto Cloud instance. E.g.: https://<tenant-id>.logto.app/',
+        type: 'string',
+      },
+      'experience-path': {
+        alias: ['path'],
+        describe: 'The local folder path of your custom sign-in experience assets.',
+        type: 'string',
+      },
+      verbose: {
+        describe: 'Show verbose output.',
+        type: 'boolean',
+        default: false,
+      },
+    }),
+  handler: async (options) => {
+    const appId = process.env.APP_ID ?? options['app-id'];
+    const appSecret = process.env.APP_SECRET ?? options['app-secret'];
+    const endpoint = process.env.ENDPOINT ?? options.endpoint;
+    const experiencePath = process.env.EXPERIENCE_PATH ?? options['experience-path'];
```

Comment:
can we try this: https://github.com/yargs/yargs/pull/308

Reply:
Updated. Env variables with a prefix `LOGTO_` will now be automatically matched and set to option values.